tools/replay.cpp

```diff
@@ -134,6 +134,8 @@ class Replay {
           replay_release();
           replay_out() << "\n";
         }
+        else if ( m_row[1] == "Umpire v0.3.2") {
```

Comment:
Hmmm.  Might want to change this to something more general.  Otherwise, we will need to update the tool for each new version.

Reply:
Good point 😄 I'll do that now.

Reply:
@mcfadden8 - this should be resolved.